### PR TITLE
quickly added events page

### PR DESCRIPTION
**Pull request by dougwoods3**
I just wanted to test this idea with you guys.

### CODE REVIEW

File: Events/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Wiki
```
Comment (albeanth) [inline Events/index.md]: ~~+title: Wiki~~

+title: Events

**Comment by albeanth** [comment]
@dougwoods3 how about we change it to something like "Upcoming Conferences"? 

This is also something we can put directly on the front page.

**Comment by albeanth** [comment]
We could do a tabular format on the front page such as:

Upcoming Conferences 
------------------------
1) ANS Summer 
2) ICTT

**Comment by dougwoods3** [comment]
I thought we might be able to use it for more than just conferences: Engineering Expo, past conferences, anything else.

**Comment by dougwoods3** [comment]
... the Engineering out loud thing.

**Comment by albeanth** [comment]
Hmmm. We could put a link for the Engineering out loud thing. 

What would we have for past stuff? What's the reason for it?

**Comment by harterj** [comment]
I like this idea. I don't see a reason for past stuff; if people want to include that work they can cite it on their own page, like, poster or presentation title from a conference. With names, I'd prefer we spells them out explicitly instead of using acronyms. That's all I have to offer

**Comment by albeanth** [comment]
@dougwoods3 I agree with @harterj on the past stuff. if people want to showcase their work from past conferences and talk etc, they can have it on their own page. 

I think having the engineering out loud podcast is good. But I think just a button/link to it on the home page will suffice.

@harterj what do you think on having the conference list on the homepage or a separate link like @dougwoods3 is proposing?

**Comment by harterj** [comment]
I think a link, like we have set up for the wiki, would be good. It would retain the "clean" feel we're  going for.

File: Events/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: Events
+layout: default
+author: Doug, Jackson, & Tony
+---
+More coming soon...
+================================
+
+# Upcoming Conferences
+* 17 January 2017 - American Nuclear Society: Oregon State University Chapter Winter Conference
```
Comment (albeanth) [inline Events/index.md]: I'm getting nit picky now.... sorry.

~~University Chapter~~ University Student Section

it's a common misnomer. they're [student sections](http://www.ans.org/const/student/) not chapters.

**Comment by albeanth** [comment]
pending @harterj 's final say, I'm good with it. :shipit:

**Comment by harterj** [comment]
Alright! @dougwoods3 and @albeanth I dig it, :shipit:

File: Events/index.md (hunk 2 above)
Comment (harterj) [reply Events/index.md]: Good eye, Tony...

**Pull request merged**